### PR TITLE
Fix jest

### PR DESCRIPTION
**Pull request by iHiD**
Happy days :)

### CODE REVIEW

File: app/javascript/components/track/approaches-elements/community-videos/CommunityVideo.tsx [hunk 1]
```diff
@@ -1,12 +1,12 @@
 import React, { useState } from 'react'
 import { Avatar, Icon } from '@/components/common'
-import { CommunityVideo } from './types'
+import { CommunityVideo as CommunityVideoType } from './types'
 import { CommunityVideoModal } from './CommunityVideoModal'
 
 export function CommunityVideo({
```
Comment (iHiD) [inline app/javascript/components/track/approaches-elements/community-videos/CommunityVideo.tsx:6]: This was redefining the type above.

File: app/javascript/utils/react-bootloader.jsx [hunk 2]
```diff
@@ -5,24 +5,28 @@ import BugsnagPluginReact from '@bugsnag/plugin-react'
...
+        return true
+      }
+
+      event.setUser(tag.content)
+    },
+  })
+  ErrorBoundary = Bugsnag.getPlugin('react').createErrorBoundary(React)
+} else {
+  ErrorBoundary = <></>
+}
```
Comment (iHiD) [inline app/javascript/utils/react-bootloader.jsx:29]: Bugsnag doesn't have an API key in test mode, so we don't initialize it, and instead effectively mock ErrorBoundary.

File: jest.config.js [hunk 3]
```diff
@@ -7,6 +7,7 @@ const config = {
       '<rootDir>/app/javascript/images/GlobalImageStub.js',
     'manifest.json$': '<rootDir>/app/javascript/__mocks__/fileMock.js',
     '@/(.*)': '<rootDir>/app/javascript/$1',
+    '\\.(css)$': 'identity-obj-proxy',
```
Comment (iHiD) [inline jest.config.js:10]: Jest was trying to parse CSS files. This stops that.

**Comment by ErikSchierboom** [comment]
Nice!

**Pull request merged**